x-pack/filebeat/module/kubernetes/k8s-ingest.yaml

```diff
@@ -0,0 +1,142 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: ingest-config
+  namespace: kube-system
+  labels:
+    k8s-app: ingest
+data:
+  filebeat-k8s.yml: |-
+    logging.level: info
+    filebeat.config:
+      modules:
+        path: ${path.config}/modules.d/*.yml
+
+    filebeat.autodiscover:
+      providers:
+        - type: kubernetes
+          hints.enabled: true
+          default.disable: true
+          
+    processors:
+      - add_kubernetes_metadata:
+          in_cluster: true
+    
+    output.elasticsearch:
+      ilm.enabled: false
+      hosts: ['${ELASTICSEARCH_HOST:elasticsearch}:${ELASTICSEARCH_PORT:9200}']
+      username: ${ELASTICSEARCH_USERNAME}
+      password: ${ELASTICSEARCH_PASSWORD}
+
+    setup.kibana:
+      host: '${KIBANA_HOST:kibana}:${KIBANA_PORT:5601}'
+---
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: ingest
+  namespace: kube-system
+  labels:
+    k8s-app: ingest
+spec:
+  template:
+    metadata:
+      labels:
+        k8s-app: ingest
+    spec:
+      serviceAccountName: ingest
+      terminationGracePeriodSeconds: 30
+      containers:
+      - name: ingest
+        image: docker.elastic.co/beats/filebeat:8.0.0
+        args: [
+          "sh", "-c", "filebeat setup -e --modules kubernetes -c /etc/filebeat-k8s.yml && filebeat -e -c /etc/filebeat-k8s.yml"
+        ]
+        env:
+        - name: ELASTICSEARCH_HOST
+          value: 192.168.99.1
```

Comment:
Should this IP be here (same for KIBANA_HOST below)? Is this a Kubernetes default for the gateway / your host?